extensions/vscode/src/views/homeView.ts

```diff
@@ -324,7 +324,7 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
       const api = await useApi();
       const apiRequest = api.files.updateFileList(
         activeConfig.configurationName,
-        uri,
+        `/${uri}`,
```

Comment:
Do you think we should be defaulting to the proper platform separator here? So we get forward or backward slashes prepended?

Reply:
The config file is expecting gitignore patterns, which always use forward slashes. We can change that if needed, but will need to convert somewhere (likely the backend).

Reply:
I think it's better to stick with slashes, because there will be cases where teams collaborate across Windows and non-Windows OSes.

Reply:
OK, I'm good with this approach. We just need to make it consistent and convert from back slashes to forward slashes when the user provides them (by manually editing the file)?